examples/remarketing/upload_conversion_adjustment.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example imports conversion adjustments for existing conversions.
+
+To set up a conversion action, run the add_conversion_action.py example.
+"""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id, conversion_action_id, gcl_id, adjustment_type,
+         conversion_time, adjustment_time, restatement_value):
```

Comment:
"conversion_time" and "adjustment_time" should be "conversion_date_time" and "adjustment_date_time" to be consistent with how it is exposed in the API (see "ConversionAdjustment").

Reply:
Looks like it has been fixed